docker_mcp/core/transfer/rsync.py

```
@@ -46,7 +46,7 @@ async def validate_requirements(self, host: DockerHost) -> tuple[bool, str]:
         try:
             result = await asyncio.get_event_loop().run_in_executor(
                 None,
-                lambda: subprocess.run(  # noqa: S603
+                lambda: subprocess.run(  # nosec B603
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)